src/sentry/tasks/auto_enable_codecov.py

```diff
@@ -23,13 +23,8 @@ def schedule_organizations(dry_run=False) -> None:
     features independently of the auto-enablement.
     """
     logger.info("Starting task for sentry.tasks.auto_enable_codecov.schedule_organizations")
-
-    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
-    logger.info(
-        "Processing organizations for codecov auto-enable", extra={"count": len(organizations)}
-    )
-    for _, organization in enumerate(
-        RangeQuerySetWrapper(organizations, step=1000, result_value_getter=lambda item: item.id)
```

Comment:
Was it batched to prevent performance issues, and turns out not needed?

Reply:
pretty much - we might still need to batch it but I don't fully understand what the batching does. I want to simplify the cron as much as possible until we get it working seamlessly, and then add back the performance improvements.